README.md

```diff
@@ -572,6 +572,11 @@ useful if you have existing machines that may have existing firewall
 configuration, and you want to make all of the firewall configuration the same
 across all of the machines.
 
+*WARNING*: When using this option, there's a small time window when firewall is
+being reset and all new connections to the system are rejected. Applying changes
+with this option in production might cause temporary service failures during the
+operation.
```

Comment:
Might be worth adding that existing connections will still pass traffic.

Reply:
please see the new text